test/tests/steel_creep_damage_oh/tests

```diff
@@ -1,40 +1,63 @@
 [Tests]
-  issues = '#231'
-  design = 'SteelCreepDamageOh.md'
-  [steel_creep_damage]
-    type = 'Exodiff'
-    input = 'steel_creep_damage.i'
-    exodiff = 'steel_creep_damage_out.e'
-    valgrind = none
-    heavy = true
+  [creepdamage]
+    issues = '#231'
+    design = 'SteelCreepDamageOh.md'
     requirement = 'Blackbear shall compute steel creep damage considering stress triaxiality as per '
-                  'Oh et al (2011) and apply it to reduce the finite element stress of a simple '
-                  'problem to simulate creep failure.'
+                  'Oh et al (2011)'
+    [steel_creep_damage]
+     type = 'Exodiff'
+      input = 'steel_creep_damage.i'
+      exodiff = 'steel_creep_damage_out.e'
+      valgrind = none
+      heavy = true
+      detail = 'Blackbear shall apply steel creep damage reduce the finite element stress of a simple '
```

Comment:
Typically the 'detail' part of the requirement can be read continuously, i.e. requirement + detail describe a complete requirement.

Reply:
Oh yeah, good catch @recuero. @albiedahal, please look at other requirements that are set up using requirement/detail for an example and copy how they did that, and then fix that other minor issue I pointed to in your .i file while you're at it.

Reply:
I fixed the issue. I also changed one of the new tests to `heavy` as per @recuero 's suggestion earlier.